docs_user/modules/proc_configuring-a-ceph-backend.adoc

```diff
@@ -1,40 +1,30 @@
 [id="configuring-a-ceph-backend_{context}"]
 
-= Configuring a Ceph backend
+= Configuring a {Ceph} back end
 
-If the original deployment uses a Ceph storage backend for any service
-(e.g. {image_service_first_ref}, {block_storage_first_ref}, {compute_service_first_ref}, {rhos_component_storage_file_first_ref}), the same backend must be used in the
-adopted deployment and custom resources (CRs) must be configured accordingly.
-
-If you use {rhos_component_storage_file_first_ref}, on {OpenStackPreviousInstaller} environments, the CephFS driver in {rhos_component_storage_file} is configured to use
-its own keypair. For convenience, modify the `openstack` user so that you
-can use it across all {rhos_prev_long} services.
-
-Using the same user across the services serves two purposes:
-
-* The capabilities of the user required to interact with {rhos_component_storage_file}
-became far simpler and hence, more became more secure with {rhos_acro} {rhos_curr_ver}.
-* It is simpler to create a common ceph secret (keyring and ceph config
-file) and propagate the secret to all services that need it.
+If your {rhos_prev_long} ({OpenStackShort}) {rhos_prev_ver} deployment uses a {Ceph} back end for any service, such as {image_service_first_ref}, {block_storage_first_ref}, {compute_service_first_ref}, or {rhos_component_storage_file_first_ref}, you must configure the custom resources (CRs) to use the same back end in the {rhos_long} {rhos_curr_ver} deployment.
 
 [TIP]
-To run `ceph` commands, you must use SSH to connect to a Ceph
-storage node and run `sudo cephadm shell`. This brings up a ceph orchestrator
-container that allows you to run administrative commands against the ceph
-cluster. If you deployed the ceph cluster by using {OpenStackPreviousInstaller}, you may launch the `cephadm` shell from an {OpenStackShort} controller node.
+If you use {rhos_component_storage_file} on {OpenStackPreviousInstaller} environments, the CephFS driver in {rhos_component_storage_file} is configured to use its own key pair. For convenience, modify the `openstack` user so that you
+can use it across all {OpenStackShort} services.
 
+[NOTE]
```

Comment:
Not sure this should be a Note or the "procedure" that should be followed to update the openstack keyring.

Reply:
Should the note that starts with "To run ceph commands..." be a prerequisite, or the first step of the main procedure?

Reply:
This command is not strictly connected to the order of services redeploy in the ctlplane, it can be treated as a prerequisite if Manila is part of the overcloud in the src cloud.
We can probably move it as a prerequisite rather then keeping it as a Note, and highlight that if Manila is part of the deployment, the openstack keyring must be updated with this `<command>` for this `<reason>`

Reply:
This is the latest iteration of the introduction and prerequisites. It might be easier to see it all together in the preview: https://logserver.rdoproject.org/83/583/3b10bf2e3f08239c3c142ecc687247a528958938/github-check/adoption-docs-preview/2fb2d12/docs_build/docs_build/adoption-user/index-downstream.html#configuring-a-ceph-backend_migrating-databases

Reply:
Ok, thank you for sharing the preview. 
Definitely I think it's good to have this step as a prerequisite.